README.dev-quickstart.md

```diff
@@ -0,0 +1,35 @@
+
+# hello world
+- toto je priznanie digital
+- vsetku logiku najdes rozsekanu po featurach v adresari "src/features"
+- tam su veci zgroupovane po namespacoch "src/features/NAMESPACE/"
+- a pod nimi su potom featury "src/features/NAMESPACE/FEATURE/"
+- pod featurou su vzdy subory k danej feature
+
+
+# ako zacat
+- npm run dev
+
+
+# ako otestovat ci si nic nerozbil (CYPRESS ui e2e testy)
+- npm run cy:run
+
+
+# ako otestovat ci si nic nerozbil (JEST unit testy)
+- npm run test
+
+
+# dev faq
+- nejde mi node, hadze chyby
+    skus node 16, nainstalujes cez nvm a potom by malo ist, s novsim su issues
```

Comment:
Preco su s novsim issues? v nvmrc a v packge.json je specifikovana verzia 16, takze pre to sa to stazuje na inych verziach, ale nenazval by som to issues, it's a feature :) Kludne to ale mozeme teraz bumpnut na Node 18, co je current LTS

Reply:
s 18 to ide, aj testy presli, ale tu na githube nepresli prave kvoli zvysenej verzii na 18